lib/msal-browser/docs/accounts.md

```diff
@@ -6,10 +6,32 @@
+
+| Property Name        | Description                                                                                                                                                                                  | Example                                                                   |
+| -------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------- |
+| `homeAccountId`      | Home account identifier for this account object in the corresponding authentication scheme. For AAD/MSA, the format is \<uid>.\<utid>, which identifies the account uniquely across tenants. | 00000000-0000-0000-0000-000000000000.72f988bf-86f1-41af-91ab-2d7cd011db47 |
+| `environment`        | Entity which issued the token represented by the domain of the issuer.                                                                                                                       | login.microsoftonline.com                                                 |
+| `tenantId` / `realm` | Full tenant or organizational id that this account belongs to.                                                                                                                               | 72f988bf-86f1-41af-91ab-2d7cd011db47                                      |
+| `username`           | The `preferred_username` claim of the id_token that represents this account                                                                                                                  | test@example.com                                                          |
+| `localAccountId`     | Local, tenant-specific account identifer for this account object, usually used in legacy cases                                                                                               | 00000000-0000-0000-0000-000000000000                                      |
+| `name`               | Full name for the account, including given name and family name.                                                                                                                             | John Doe                                                                  |
+| `nativeAccountId`    | The user's native account ID                                                                                                                                                                 | 00000000-0000-0000-0000-000000000000                                      |
+
+> Note: `realm` is the name that `tenantId` receives in the cache.
+
+The following `getAccountBy` APIs are marked for deprecation and will be removed in a future version of MSAL. Please migrate to `getAccountByFilter()`:
```

Comment:
I assume you added `@deprecated` tag for typedocs too.

Reply:
Yep you can verify that in the diff for PublicClientApplication.